configs/rhel-sst-cockpit--web-console.yaml

```diff
@@ -6,16 +6,14 @@ data:
   maintainer: rhel-sst-cockpit
   labels:
     - eln
-    - c10s
+    - c11s
```

Comment:
`c11s` is not yet a valid label.  The way you split this, just drop cNs labels and leave it `eln` only.  When C11S is branched, then all workloads with an `eln` label will have a `c11s` label added.
```suggestion
```

Reply:
Ack! updated.